### ISSUE TITLE
Latest install (snap) on Ubuntu not working...

### ISSUE THREAD

**Issue by HakkaTjakka** [OpenRA/OpenRA#20795]
## Issue Summary

When the game starts you can not move or deploy the MCV... Then you'r stuck.

## Reproduction

Install game with snap on Ubuntu Linux. Start. Play. --> Can not move or deploy...

## Expected behavior

Move and/or deploy MCV.

**Comment by ThomasChr** [comment]
Installed `Ubuntu 22.04.2 LTS` (minimal) installed Gnome Desktop Environment, installed xrdp and OpenRA (`snap install openra`).
No problems, I can deploy the MCV, I can build buildings and units and also move them.
So can't recreate, sorry.

**Comment by anvilvapre** [comment]
Sounds like a mouse issue. 

Does your mouse move/scroll? Can you draw a select box? 

Depending on your window manager. Try `alt+tab`. Perhaps the OpenRA window lost mouse focus due to some notification popup window.

**Comment by PunkPun** [comment]
We do know quite a few people play on Ubuntu and they haven't experienced such issues. It may be a specific setup problem

**Comment by anvilvapre** [comment]
yes, i play on Ubuntu. just that i do not use snap, but the app image.

**Comment by PunkPun** [comment]
Closing as outdated
